AWS/legos/aws_get_redshift_query_details/README.md

```diff
@@ -0,0 +1,29 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h2>Get Datadog Handle</h2>

```

Comment:
Heading of the AWS/legos/aws_create_redshift_query/README.md file is same for AWS/legos/aws_get_redshift_query_details/README.md